ios/Podfile

```diff
@@ -15,6 +15,8 @@ target 'atb' do
   permissions_path = '../node_modules/react-native-permissions/ios'
   pod 'Permission-LocationWhenInUse', :path => "#{permissions_path}/LocationWhenInUse.podspec"
 
+  pod 'RNLocalize', :path => '../node_modules/react-native-localize'
```

Comment:
Because of auto-link this shouldn't be necessary

Reply:
:+1: